processor/metricstransformprocessor/README.md

```diff
@@ -281,6 +284,23 @@ operations:
     aggregation_type: sum
 ```
 
+### Split by name and labels
+```yaml
+# Convert the metric `system.cpu.usage` with the attribute `state=idle` to be a new metric named `cpu.idle`
+#
+# This can be usage for migrating from existing telemetry agents to the open telemetry collector while preserving
+# existing names used in dashboards, charts, alerts and runbook. 
+include: system.cpu.usage
+action: splitby
+new_name:  cpu.idle
+match_type: strict
+experimental_match_labels:
+  state: idle
+operations:
+  # Any operations defined here will be applied to the new metric if matched
+  ...
+```
```

Comment:
Can this be:

```
```yaml
# rename system.cpu.usage to system.cpu.usage_time
include: system.cpu.usage
action: insert
new_name: cpu.idle
experimental_match_labels:
  state: idle
operations:
  - action: aggregate_labels
    label_set: [ state ]
    aggregation_type: sum
```

Reply:
Yeah, that is also works if you want to remove the `sum` attribute from the new metric.